docs/recipes/browser-testing.md

```diff
@@ -0,0 +1,73 @@
+# Setting up AVA for browser testing.
+
+Ava doesn't come with support for the browser.
+Some libraries require that a window object is defined.
+An example of this is React.js, at least if you want to use ReactDOM.render and simulate events with ReactTestUtils.
+
+This recipe will work for any library that needs a mocked browser environment.
+
+## First step
```

Comment:
Instead of first/second/Nth step, better provide a meaningful title that indicates what this step is about. For example, "Install jsdom"